tools/system_libs.py

```diff
@@ -323,13 +323,17 @@ def create_dlmalloc(out_name):
   def create_wasm_rt_lib(libname, files):
     o_s = []
     commands = []
+    output_flag = '-c' if shared.Settings.EXPERIMENTAL_USE_LLD else '-S'
```

Comment:
a comment explaining this might help (I honestly don't know what `-S` means here)